__tests__/workers/cdc.ts

```diff
@@ -1199,6 +1203,44 @@ describe('source member', () => {
       jest.mocked(notifyMemberJoinedSource).mock.calls[0].slice(1),
     ).toEqual([base]);
   });
+
+  it('should notify when role changed', async () => {
+    const after: ChangeObject<ObjectType> = {
+      ...base,
+      role: SourceMemberRoles.Owner,
+    };
+    await saveFixtures(con, User, [defaultUser]);
+    await expectSuccessfulBackground(
+      worker,
+      mockChangeMessage<ObjectType>({
+        after,
+        before: base,
+        op: 'u',
+        table: 'source_member',
+      }),
+    );
+    expect(notifySourceMemberRoleChanged).toBeCalledTimes(1);
+    expect(
+      jest.mocked(notifySourceMemberRoleChanged).mock.calls[0].slice(1),
+    ).toEqual([base.role, after]);
+  });
+
+  it("should not notify if role doesn't change", async () => {
```

Comment:
```suggestion
  it('should not notify if role doesn't change', async () => {
```

Reply:
That won't work with the `doesn't` apostrophe right? 👀
You prefer backticks in that case?

Reply:
Oh totally missed that. Ignore me then